app/src/main/cpp/skyline/services/hid/IHidServer.cpp

```diff
@@ -179,4 +179,13 @@ namespace skyline::service::hid {
 
         return {};
     }
+
+    Result IHidServer::StartSixAxisSensor(type::KSession &session, ipc::IpcRequest &request, ipc::IpcResponse &response) {
+        return {};
+    }
+
+    Result IHidServer::GetVibrationDeviceInfo(type::KSession &session, ipc::IpcRequest &request, ipc::IpcResponse &response) {
+        response.Push(NpadVibrationDeviceInfo{0,0});
```

Comment:
It would be great if this was implemented further, reporting no vibrators here might lead to the game not submitting any vibrations.